ow_plexil/src/plans/FaultHandlingPattern2.plp

```diff
@@ -29,35 +29,32 @@ FaultHandlingPattern2: UncheckedSequence
   {
     Antenna: UncheckedSequence
     {
-      Repeat true;
+      Repeat !Lookup(PowerFault);
 
-      Pan: UncheckedSequence
+      Pan:
       {
         Start !Lookup(AntennaPanFault);
-        Exit Lookup(AntennaPanFault);
+        Invariant !Lookup(AntennaPanFault);
 
         NewAngle = (Lookup(PanDegrees) + 15) mod 360;
         if (NewAngle > 180) NewAngle = NewAngle - 360;
```

Comment:
Missing `endif`

Reply:
See comment in FaultHandlingPattern1.plp.